straxen/plugins/s2cnn_processing.py

```diff
@@ -0,0 +1,154 @@
+try:
```

Comment:
I guess you wrote this to work in some other environment but this should not go in straxen, can you remove this kind of logic?

Reply:
Yes, i can remove it. The idea was to make DoubleWidthConverter available even for environments without strax installed, but probably it is not necessary there. I think i will then place the converter to a separate file.

Reply:
I removed this logic from plugin file.